.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @evergreen-ci/dev-prod-dag
+* @10gen/dev-prod-dag
```

Comment:
You probably want `@mongodb/dev-prod-dag`. Unfortunately, the way github works, we have a separate team for each org (but it is 'dev-prod-dag' in all the orgs).